src/contribute/contribution-guide.md

```diff
@@ -244,16 +244,28 @@ All code should have appropriate unit testing coverage. New code should have
 new tests in the same contribution. Bug fixes should include a regression test
 to prevent the issue from reoccurring.
 
-#### Java SDK
+The entire set of unit tests can be run with this command at the root of the
+git repository.
 
-For contributions to the Java code, run unit tests locally via Maven.
+    $ ./gradlew test
 
-    $ mvn clean verify
+Run this command to test for lint errors.
+
+    $ ./gradlew lint
```

Comment:
I don't think this needs to be mentioned here since the lint task only lints the Python code right now.

Reply:
Removed.